named_arrays/scalars/core.py

```diff
@@ -887,6 +887,8 @@ class ScalarArray(
     def __post_init__(self: Self):
         if self.axes is None:
             self.axes = tuple()
+        if type(self.axes) is str:
+            self.axes = (self.axes,)
```

Comment:
This looks good for now. The only problem is if someone sets `ScalarArray.axes` after the array has already been constructed, but we have that problem already if `len(ScalarArray.axes) != np.ndim(ScalarArray.ndarray)` so we'll have to handle that in a separate PR.